x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/actions/index/ExecutableIndexAction.java

```diff
@@ -145,6 +146,22 @@ public ExecutableIndexAction(IndexAction action, Logger logger, Client client,
             }
             bulkRequest.add(indexRequest);
         }
+
+        if (ctx.simulateAction(actionId)) {
+            try (XContentBuilder builder = jsonBuilder().startArray()) {
```

Comment:
Can `BulkItemResponse` be reused here and then serialise that to json?
This way if the format ever changes then this code doesn't need to be changed.

Reply:
there is no bulk item response, because the request is not sent but only simulated

Reply:
Right, my comment doesn't make much sense. Most of the `BulkItemResponse` instance would have dummy values (except for id and index).